db/controllers/controllers.js

```diff
@@ -14,19 +19,29 @@ exports.GetTopics = (req, res, next) => {
 }
 
 
-exports.getArticleById = (req, res,next) => {
-const { article_id } = req.params;
+exports.getArticleById = (req, res, next) => {
+    const { article_id } = req.params;
     selectArticleById(article_id).then((article) => {
-    res.status(200).send({ article })
+        res.status(200).send({ article })
     })
         .catch(next)
-
+}
 
 exports.GetArticles = (req, res, next) => {
   fetchArticles()
     .then((articles) => {
       res.status(200).send({ articles});
     })
     .catch(next);
+};
+
+exports.getCommentById = (req, res, next) => {
+    const { article_id } = req.params;
+    const { sort_by } = req.query;
+    fetchArticalsByAricle_id(article_id, sort_by)
```

Comment:
Please make sure, you only send an empty array of comments when the article_id exists in the database. For this you would need to check both.